master/usage/migrating-pools.md

```diff
@@ -0,0 +1,162 @@
+---
+title: Migrating to a new IP Pool
+---
+
+## About migrating active pools
+
+When using {{ site.prodname }} IPAM, each workload is assigned an address from the selection of configured pools.
+There are a few reasons you may want to change the pool being used on an active cluster, including moving to a
+larger CIDR which can accomodate more workloads, or moving off of a CIDR that was used accidentally.
+
+### Purpose of this page
+
+Provide guidance on how to migrate off of a live {{ site.prodname }} pool.
+
+## Prerequisites
+
+**{{site.prodname}} IPAM**
+
+This guide only applies if you are using {{site.prodname}} IPAM.
+
+**Orchestrator support**
+
+While {{ site.prodname }} supports changing the live address range, not all orchestrators do.
+Be sure to consult the documentation for whichever installer you are using to ensure it supports changing it.
+
+For example, in Kubernetes, all three of the following arguments must be equal to, or contain, the Calico
+IP Pool CIDRs:
+
+- kube-apiserver's `--pod-network-cidr`
+- kube-proxy's `--cluster-cidr`
+- kube-controller-manager's `--cluster-cidr`
+
+Another example is OpenShift, which does not support changing the pod network CIDR (as per their [documentation on the `osm_cluster_network_cidr` configuration variable](https://docs.openshift.org/latest/install_config/install/advanced_install.html#configuring-cluster-variables).
+
+**Application availability impact**
+
+This process will require the recreation of all {{ site.prodname }}-networked workloads, which will have some impact on
+the availability of your applications.
+
+## Consequences of deleting of an active Pool without following this migration procedure
+
+Removing an active pool without following the migration procedure can cause network connectivity disruptions in any running
+workloads with addresses from that pool. Namely:
+
+- If IP-in-IP was enabled on the pool, those workloads will no longer have their traffic encapsulated.
+- If nat-outgoing was enabled on the pool, those workloads will no longer have their traffic NAT'd.
+- If using Bird for routing, routes to pods will no longer be aggregated.
```

Comment:
We should try to avoid talking about component specifics - better to say "If using Calico BGP routing" or something like that.